.eslintrc.js

```diff
@@ -1,7 +1,7 @@
-{
-  "plugins": ["@typescript-eslint"],
-  "extends": ["next", "plugin:@typescript-eslint/recommended", "prettier"],
-  "rules": {
+module.exports = {
```

Comment:
Bonus: change file extension to `*.js` to enable Prettier and to allow comments (which we'll probably need in the future)